docs/docs/contributing/development.mdx

```diff
@@ -219,22 +219,22 @@ If you have made changes to the FAB-managed templates, which are not built the s
 If you add a new requirement or update an existing requirement (per the `install_requires` section in `setup.py`) you must recompile (freeze) the Python dependencies to ensure that for CI, testing, etc. the build is deterministic. This can be achieved via,
 
 ```bash
-$ python3 -m venv venv
-$ source venv/bin/activate
-$ python3 -m pip install -r requirements/development.txt
-$ pip-compile-multi --no-upgrade
+python3 -m venv venv
```

Comment:
I suppose some people like the prompts in their doc... but I'm personally fine without them.

Reply:
Well, that fixes a Markdown warning: `Dollar signs used before commands without showing output`.
If someone needs to copy this into their terminal, they have to delete a lot of dollar signs.